nexus-writer/src/main.rs

```diff
@@ -192,7 +205,6 @@ async fn main() -> anyhow::Result<()> {
         metrics::Unit::Count,
         "Number of failures encountered"
     );
-
     let run_ttl =
```

Comment:
```suggestion

    let run_ttl =
```

Reply:
Please avoid removing whitespace like this, it makes things that are mostly unrelated seem more related than they are.